website/content/en/docs/User Guide/Learn Cloud Operations/cloud_error_reporting.md

```diff
@@ -37,4 +38,50 @@ Click **View Logs** for one of the samples to see the log messages that match th
 
 You can expand any of the messages that matches the filter to see the full stack trace:
 
-![image](/docs/images/user-guide/35-logdet.png)
\ No newline at end of file
+![image](/docs/images/user-guide/35-logdet.png)
+
+#### Errors Manufacturing 
+
+There are several ways in which you can experiment with Error Reporting tool and manufacture errors that will be reported and displayed in the UI. For the purpose of this demonstration we will use 2 tools that are coming with Cloud Operations Sandbox: Load Generator and SRE Recipes to simulate a situation that `Sandbox break`.
+
+To simulate requests using the load generator we can use the UI or `sandboxctl`
+
+```
+$sandboxctl loadgen step
+Redeploying Loadgenerator...
+Loadgenerator deployed using step pattern
+Loadgenerator web UI: http://<ExampleIP>
+```
+
+Then to `break` the service we will use sre-recipes (recipe2)
+
+```
+$sandboxctl sre-recipes break recipe2
+Breaking service operations...
+...done
+```
+
+In this case you will see in Error Reporting UI you will see a new reported error `Unhealthy pod, failed probe`
+
+![image](/docs/images/user-guide/51-Error-Reporting-podfailed.png)
+
+You can open it to see additional information, in the below example you can see that this error repeat itself several times in the last hour.
+
+![image](/docs/images/user-guide/52-Error-Reporting-pod.png)
+
+You can also press `View logs` to view detailed log information in Cloud Operations Logging.
+  
+![image](/docs/images/user-guide/53-Error-Reporting-logs.png)
+
+> Note: at the end, don't forget to recover the service using `sandboxctl sre-recipes restore`. 
+Another way to `break the service` is to use the load generator to overload the service with too many requests.
```

Comment:
I'd probably remove the part about breaking the service using the load generator.  It seems a bit out of place in the Note here.

If we wanted to include it, we should probably pull it out of the note and give it more context. But I think the sre-recipe stuff is enough

Reply:
A new line was missing. 
I added it to show them that can do things manually outside the scope of the tool, do you have a preferable alternative ?

Reply:
I'd personally just leave that part out and stick to sandboxctl. Ideally at some point we'll refactor the services to not fail under high load, so we probably shouldn't advertise this as a feature.